.github/workflows/release.yml

```diff
@@ -29,6 +29,14 @@ jobs:
           plutil -replace CFBundleVersion -string ${{ github.event.inputs.version }} -- 'Demo/Application/Supporting Files/Braintree-Demo-Info.plist'
           plutil -replace CFBundleVersion -string ${{ github.event.inputs.version }} -- 'Sources/BraintreeCore/Info.plist'
 
+      # Remove the SPMTest app to prevent Carthage timeout; store SPMTest app in temp dir
```

Comment:
Maybe we should do this before the `Update version` step. `Update version` and `Tag and push new version` kind of go together, since once is updating a bunch of files and the other is committing those changes.

Reply:
DONE